Sources/EdgeNetworkHandlers/EdgeEndpoint.swift

```diff
@@ -60,7 +60,7 @@ struct EdgeEndpoint {
         guard let domain = domain, !domain.isEmpty else {
             return EdgeConstants.NetworkKeys.EDGE_DEFAULT_DOMAIN
         }
-        return domain.lowercased().deletePrefix("https://").deletePrefix("http://")
+        return domain.lowercased().deletePrefix("https://").deletePrefix("http://").deleteSuffix("/")
```

Comment:
Should we also cover the use-case where the slash character is present multiple times, like `my.domain///` ?
I am somewhat more inclined to have this cleanup solved server-side and in the Launch extension. 
What do you think?